docs/source/pages/supported_technologies/index.rst

```diff
@@ -86,8 +89,9 @@ Provenances
      - Documentation
    * - `SLSA <https://slsa.dev>`_
      -
-       * `SLSA provenance version 0.2 <https://slsa.dev/spec/v0.2/provenance>`_.
-       * The provenance should be published as a GitHub release asset
+       * | `SLSA provenance version 0.2 <https://slsa.dev/spec/v0.2/provenance>`_. The provenance should be
```

Comment:
I think we should explicitly say that the provenance can `either` be  `published as a GitHub release asset.` or `published on the `npm registry <https://registry.npmjs.org>`_.`